src/routes/routes.js

```diff
@@ -45,4 +46,9 @@ router.delete('/api/v1/profiles/:userName/unfollow', verifyToken, UserFollow.unF
 router.get('/api/v1/profiles/:userName/following', verifyToken, UserFollow.getFollowingList);
 router.get('/api/v1/profiles/:userName/followers', verifyToken, UserFollow.getFollowersList);
 
+router.patch('/api/v1/profiles/:username/notification/:subscribe', [verifyToken, canEditProfile], Notification.optInOutNotificatation);
```

Comment:
by convention, the resource part has to be in plural i.e. `notifications` not `notification`. Otherwise looks good

Reply:
Thank you, that's right.

Reply:
Done